server/integration/build/src/main/resources/modules/system/layers/base/org/hibernate/search/engine/main/module.xml

```diff
@@ -15,6 +15,7 @@
         <module name="org.hibernate.commons-annotations" />
         <module name="org.infinispan" />
         <module name="org.infinispan.lucene-directory" />
```

Comment:
I guess we should be able to drop the dependency to 'org.infinispan' and 'org.infinispan.lucene-directory' soon?
And also to re-use the modules definition from the Hibernate Search release rather than using a custom one.